### PR TITLE
[telemetry] Add telemetry to SbPlayer DRM

### PR DESCRIPTION
**Pull request by borongc**
Adding additional Media Telemetry to cover SbPlayer DRM functions.

The rough range of each function:
1. SbDrmDestroySystem(): 3.77ms to 111.17ms
2. SbDrmGenerateSessionUpdateRequest(): 32.33ms to 338.9ms
3. SbDrmUpdateSession(): 13.29ms to 267.52ms
4. SbDrmCloseSession(): 2.9ms to 23.71ms

b/303328238

### CODE REVIEW

**Review by at-ninja — APPROVED** [review]

File: cobalt/media/base/metrics_provider.cc [hunk 1]
```diff
@@ -151,6 +151,27 @@ void MediaMetricsProvider::ReportActionLatencyUMA(
       UMA_HISTOGRAM_TIMES("Cobalt.Media.SbPlayer.Destroy.LatencyTiming",
                           action_duration);
       break;
+    case MediaAction::SBDRM_DESTROY_SYSTEM:
+      UMA_HISTOGRAM_CUSTOM_MICROSECONDS_TIMES(
+          "Cobalt.Media.SbDrm.Destroy.LatencyTiming", action_duration,
+          base::TimeDelta::FromMicroseconds(500),
```
Comment (joeltine) [inline cobalt/media/base/metrics_provider.cc:157]: I believe this is quite low compared to the 3.77ms estimate you give in the PR description

File: cobalt/media/base/metrics_provider.cc [hunk 2]
```diff
@@ -151,6 +151,27 @@ void MediaMetricsProvider::ReportActionLatencyUMA(
...
+          base::TimeDelta::FromMilliseconds(250), 50);
+      break;
+    case MediaAction::SBDRM_GENERATE_SESSION_UPDATE_REQUEST:
+      UMA_HISTOGRAM_TIMES(
```
Comment (joeltine) [inline cobalt/media/base/metrics_provider.cc:161]: This histogram ranges from 1ms to 10s. The upper bound in particular is way higher than your estimate. FYI. You ok with this?  Same with your other histograms below.

Comment (borongc) [reply cobalt/media/base/metrics_provider.cc:161]: Yes, as the average is about ~173ms for `SbDrmGenerateSessionUpdateRequest()`, it is expected that the histogram would look like `SbPlayerDestroy()` in https://github.com/youtube/cobalt/pull/3822.

As it is possible that there are many other slower devices, keeping it in the range of 1ms to 10s can ensure we can capture the distribution.

File: cobalt/media/base/metrics_provider.cc (hunk 1 above)
Comment (borongc) [reply cobalt/media/base/metrics_provider.cc:157]: Yes, this is an extra margin for the lower bound.

**Review by joeltine — APPROVED** [review]

**Pull request merged**